include/SDL_mixer.h

```diff
@@ -620,6 +621,10 @@ extern DECLSPEC int SDLCALL Mix_PausedMusic(void);
  */
 extern DECLSPEC int SDLCALL Mix_ModMusicJumpToOrder(int order);
 
+/* Tracks */
+extern DECLSPEC void SDLCALL Mix_StartTrack(int track);
+extern DECLSPEC int SDLCALL Mix_GetNumTracks(Mix_Music *music);
+
```

Comment:
Any thoughts on these APIs/naming?

Reply:
The naming seems fine.

@sezero, does this fit in with any of the other supported music formats?
@icculus, do you have any thoughts on this PR?

Reply:
> @sezero, does this fit in with any of the other supported music formats?

No, I don't think so.